labs/iot/gps-lab/steps/add-gps-to-pi.md

```diff
@@ -0,0 +1,19 @@
+# Add the GPS sensor to Raspberry Pi
+
```

Comment:
In all the steps can we start with a review of the previous step (if applicable), then an intro to the current step. See https://github.com/microsoft/iot-curriculum/blob/main/labs/iot/environment-monitor/steps/set-up-pi.md#set-up-a-raspberry-pi-to-send-temperature-data for an example.